services/frontend-service/src/legacy-ui/AppBar/Header.tsx

```diff
@@ -22,15 +22,21 @@ import { AppDrawer } from './AppDrawer';
 import * as api from '../../api/api';
 import { LocksDrawer } from './LocksDrawer';
 import { ActionsCart } from '../ActionsCart/ActionsCart';
+import Tooltip from '@material-ui/core/Tooltip';
 
-const Header: React.FC<any> = (props: { overview: api.GetOverviewResponse }) => {
-    const { overview } = props;
+const Header: React.FC<any> = (props: {
+    overview: api.GetOverviewResponse;
+    configs: api.GetFrontendConfigResponse;
+}) => {
+    const { overview, configs } = props;
     return (
         <AppBar>
             <Box sx={{ display: 'flex' }}>
                 <Typography component="h1" variant="h6" color="inherit" noWrap sx={{ flexGrow: 1, width: '24rem' }}>
                     <strong>
-                        <code>KUBERPULT UI</code>
+                        <Tooltip title={`Kuberpult ${configs.version || ''}`}>
```

Comment:
would this show up as an empty tooltip? if so might want to make the default value (if no version is supplied) the same as the text
```suggestion
                        <Tooltip title={`Kuberpult ${configs.version || 'KUBERPULT UI'}`}>
```

Reply:
NO! it will not be an empty tooltip if there is no version supplied it will be `Kuberpult`